src/api/cli/address_gen.go

```diff
@@ -15,7 +18,7 @@ func addressGenCmd() gcli.Command {
 		Description: "",
 		Flags: []gcli.Flag{
 			gcli.IntFlag{
-				Name:  "count,c",
+				Name:  "n",
```

Comment:
We need to retain support for "count,c" or prepare for release "breaking changes" document, since this is an API change and exchanges must be notified.